src/pages/home/report/ContextMenu/ContextMenuUtils/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable import/prefer-default-export */
+
+/**
+ * We should show popover description only on mWeb
+ *
+ * @param {String} selection
+ * @param {Boolean} isSmallScreenWidth
+ * @returns {Boolean}
```

Comment:
JS docs need updated now in both modules (eg. `@returns` is wrong) and the method description should probably say something like: 

> The popover description will be an empty string on anything but mobile web because... (I'm not actually 100% why)